students/becky_larson/lesson05/Mailroom_Part3.py

```diff
@@ -136,25 +139,39 @@ def create_report(donors):
     """
 
     print("** You've selected to create a report.  **\n")
+    report = []
+    
     col1 = 'Donor Name'
     col2 = 'Total Given'
     col3 = 'Num Gifts'
     col4 = 'Average Gift'
-    print(f'{col1:25} | {col2:13} | {col3:11} | {col4:13}')
-    print('-'*70)
+    # print(f'{col1:25} | {col2:13} | {col3:11} | {col4:13}')
+    # print('-'*70)
+    report.append(f'{col1:25} | {col2:13} | {col3:11} | {col4:13}')
+    report.append('-'*70)
 
     donors = dict(sorted(donors.items(), key=sort_key, reverse=True))
     for donor, donations in donors.items():
         count = len(donations)
         total = sum(donations)
         avg = total/count
-        print(f'{donor:25}  ${total:13.2f}   {count:11}  ${avg:12.2f}')
-    return
+        # print(f'{donor:25}  ${total:13,.2f}   {count:11}  ${avg:12,.2f}')
+        report.append(f'{donor:25}  ${total:13,.2f}   {count:11}  ${avg:12,.2f}')
+    report = '\n'.join(report)
+    print(report)
+    print("\n** Thank you!  **\n")
 
 
 def ask_for_folder():
     global user_folder
 
+    # as creating folders is platform specific, end if not Windows. 
+    import platform
```

Comment:
your import should go at the top of the file (exceptions are when you have to deal with circular imports which is a different topic)